### PR TITLE
Support OPTIMIZELY_DATA_DIR envar + some cleanup

### PR DESCRIPTION
**Pull request by koomen**

### CODE REVIEW

File: labs/query-enriched-event-data-with-spark/index.md [hunk 1]
```diff
@@ -25,19 +25,56 @@ spark = SparkSession \
...
+
+We'll load decision data from the `type=decisions` directory in the base data directory.
+
+
+```python
+decisions_dir = os.path.join(base_data_dir, "type=decisions")
+read_data(decisions_dir, "decisions")
+```
+
+    Reading decisions data from data/type=decisions
```
Comment (asaschachar) [inline labs/query-enriched-event-data-with-spark/index.md:64]: Is this intended to be a header for the content above?

File: labs/query-enriched-event-data-with-spark/index.md [hunk 2]
```diff
@@ -25,19 +25,56 @@ spark = SparkSession \
     .builder \
     .appName("Python Spark SQL") \
     .config("spark.sql.repl.eagerEval.enabled", True) \
+    .config("spark.sql.repl.eagerEval.truncate", 120) \
     .getOrCreate()
 ```
 
 ## Loading enriched event data
```
Comment (asaschachar) [inline labs/query-enriched-event-data-with-spark/index.md:32]: nit capitalization: Enriched Event data

File: labs/query-enriched-event-data-with-spark/index.md [hunk 3]
```diff
@@ -25,19 +25,56 @@ spark = SparkSession \
...
 
-We'll start by loading decision and conversion data from `/data`.
+The `OPTIMIZELY_DATA_DIR` environment variable may be used to specify the local directory where Enriched Event Data is stored.  If, for example, you've downloaded Enriched Event data and saved it in `optimizely_data` in your home directory, you can load that data in this notebook by executing the following command before launching Jupyter Lab:
```
Comment (asaschachar) [inline labs/query-enriched-event-data-with-spark/index.md:34]: nit: Enriched Event **d**ata

File: labs/query-enriched-event-data-with-spark/index.md (hunk 1 above)
Comment (koomen) [reply labs/query-enriched-event-data-with-spark/index.md]: No; this was `print` output from the `read_data` function.  I agree it looks confusing in the markdown file, so I removed it from the notebook.

File: labs/query-enriched-event-data-with-spark/index.md (hunk 2 above)
Comment (koomen) [reply labs/query-enriched-event-data-with-spark/index.md]: Fixd

File: labs/query-enriched-event-data-with-spark/index.md (hunk 3 above)
Comment (koomen) [reply labs/query-enriched-event-data-with-spark/index.md]: Fixed.

**Pull request merged**